cmd/openshift-install/create.go

```diff
@@ -235,8 +235,7 @@ func destroyBootstrap(ctx context.Context, directory string) (err error) {
 		},
 	)
 	if err != nil {
-		logrus.Error(errors.Wrap(err, "waiting for bootstrap-complete"))
-		return nil
+		return errors.Wrap(err, "waiting for bootstrap-complete")
```

Comment:
to compare it to how the error would have been structured without wrap
`fmt.Errorf("failed waiting for bootstrap-complete: %v", err)`

I would suggest we keep it `errors.Wrap(err, "failed waiting for bootstrap-complete")`